lib/course_work_courses.rb

```diff
@@ -35,7 +35,7 @@ def cross_listings
   end
 
   def self.instance
-    # @instance ||= CourseWorkCourses.new
+    # This used to be memoized, i.e. ||= but we want the JSON loaded each time without restarting
```

Comment:
Can we get rid of the `instance` method then?  The name of this method sort of indicates this is the "Singleton Pattern".  But now that this isn't memoized, it's not a Singleton.

Reply:
Sure